APMrover2/sailboat.cpp

```diff
@@ -86,6 +86,24 @@ const AP_Param::GroupInfo Sailboat::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("WNDSPD_MIN", 7, Sailboat, sail_windspeed_min, 0),
 
+    // @Param: SAIL_MAX_X_TRACK
+    // @DisplayName: Sailing vehicle max cross track error
+    // @Description: The sail boat will tack when it reaches this cross track error, defines a corridor of 2 times this value wide, 0 disables
+    // @Units: m
+    // @Range: 5 25
+    // @Increment: 1
+    // @User: Standard
+    AP_GROUPINFO("MAX_X_TRACK", 7, Sailboat, max_cross_track, 10),
+
+    // @Param: SAIL_LOIT_RADIUS
```

Comment:
we can leave off the "SAIL_" part of this parameter as well I think

Reply:
fixed